src/app/components/Messages/PrivateMessageModal.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="message panel" tabindex="0">
+    <span id="user">To: {{username}}</span>
+    <textarea type="text" id="msg" name="msg" ref="ta" v-model="msg"/>
+    <button
+      class="btn"
+      style="width: 100%; left: 50%; margin-top:5px"
+      @click="send"
+    >
+      Continue
+    </button>
+    <button
+      class="btn"
+      style="width: 100%; left: 50%; margin-top:10px;"
+      @click="cancel"
+    >
+      Cancel
+    </button>
+  </div>
+</template>
+
+<script lang="ts">
+  /* Taken from OperLogin and modified */
+  import { Component, Vue } from 'vue-property-decorator';
+  import Message from '@/types/message';
+
+
+  @Component({
+    components: {},
+  })
+  export default class PrivateMessageModal extends Vue {
+    msg = 'Message';
+
+    get username() {
+      return this.$vxm.chat.userToPrivMsg;
+    }
+
+    onStart() {
+      this.$nextTick(() => this.$refs.ta.focus());
+      this.$refs.ta.setSelectionRange(0, this.$refs.ta.value.length);
+    }
+
+    $refs !: {
+      ta: HTMLTextAreaElement,
+    };
```

Comment:
1) rename `ta` to `textArea` or something like that
2) `@Ref() readonly textArea!: HTMLTextAreaElement;` is a thing you can do (you import `Ref` alongside the other attributes)